components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildValidationDataFields.java

```diff
@@ -9,7 +9,7 @@
 import java.util.function.Function;
 import java.util.stream.Stream;
 
-public enum Fields {
+public enum BuildValidationDataFields {
```

Comment:
I named this class `BuildValidationDataFields` since it contains the fields of `BuildValidationData` which already existed.

This allowed me to follow the same convention for `BuildTimeMetrics` and `BuildTimeMetricsFields`.

I'm fine changing it, but we should probably also rename `BuildValidationData`.

Reply:
I updated `BuildValidationDataFields` -> `BuildScanDataFields` in 499b9e75414e49a17833b8cda17a0eb603efd821

The question is still open as to also rename `BuildValidationData` and `FetchBuildValidationDataCommand`.

Reply:
Renamed in c32ab92b4ab9c5201a0c732cb3cf0164e1dbda93

Reply:
LGTM, except to method renaming suggestions.

Reply:
Great catch. Fixed in d8d9203f6898a5b752c888e77339cbed23813389.

Reply:
LGTM